### PR TITLE
https://issues.redhat.com/browse/ACM-5962 Known issue pruneObjectBeha…

### PR DESCRIPTION
**Pull request by dockerymick**
…vior
https://issues.redhat.com/browse/ACM-5962

### CODE REVIEW

File: release_notes/known_issues_governance.adoc [hunk 1]
```diff
@@ -78,10 +78,15 @@ oc get configurationpolicy -n <cluster-namespace> -o name | xargs oc patch -n <c
...
 //2.6:25261
+//2.7+2.8:5939
 
-When modifying an existing configuration policy, `DeleteAll` or `DeleteIfCreated` in the `pruneObjectBehavior` feature does not clean up old resources that were created before modifying. Only new resources from policy creations and policy updates are tracked and deleted when you delete the configuration policy.
+When you modify an existing configuration policy, `pruneObjectBehavior` does not work. View the following reasons why `pruneObjectBehavior` might not work:
+
+- If you set `pruneObjectBehvior` to `DeleteAll` or `DeleteIfCreated`, old resources that were created before modifying are not cleaned correctly. Only new resources from policy creations and policy updates are tracked and deleted when you delete the configuration policy.
+
+- If you set `pruneObjectBehvior` to `None` or fail to set the parameter value, old objects might be unintentionally deleted on the managed cluster. Specifically, this occurs when a user changes the `name`, `namespace`, `kind`, or `apiversion` in the template. The parameter fields can dynamically change when the `object-templates-raw` or `nameSelector` parameters change.
```
Comment (yiraeChristineKim) [inline release_notes/known_issues_governance.adoc]: should be `namespaceSelector`?!!

Comment (yiraeChristineKim) [inline release_notes/known_issues_governance.adoc]: To me, **fail to set the parameter value** unclear, what does the **parameter value** indicate?

Comment (yiraeChristineKim) [reply release_notes/known_issues_governance.adoc]: is that `name`, `namespace`, `kind`, or `apiversion`?

**Review by JustinKuli — APPROVED** [review]
Sounds good to me.

File: release_notes/known_issues_governance.adoc (hunk 1 above)
Comment (JustinKuli) [inline release_notes/known_issues_governance.adoc]:
```suggestion
- If you set `pruneObjectBehvior` to `None` or fail to set the parameter value, old objects might be unintentionally deleted on the managed cluster. Specifically, this occurs when a user changes the `name`, `namespace`, `kind`, or `apiversion` in the template. The parameter fields can dynamically change when the `object-templates-raw` or `namespaceSelector` parameters change. 
```

@yiraeChristineKim is keeping me honest and noticed this typo I missed

**Review by JustinKuli — CHANGES_REQUESTED** [review]

File: release_notes/known_issues_governance.adoc [hunk 2]
```diff
@@ -78,10 +78,15 @@ oc get configurationpolicy -n <cluster-namespace> -o name | xargs oc patch -n <c
...
 [#pruneobjbeh-not-working-existing-config-policy]
-== PruneObjectBehavior does not work when modifying existing configuration policy
+== _pruneObjectBehavior_ does not work when modifying existing configuration policy
 //2.6:25261
+//2.7+2.8:5939
 
-When modifying an existing configuration policy, `DeleteAll` or `DeleteIfCreated` in the `pruneObjectBehavior` feature does not clean up old resources that were created before modifying. Only new resources from policy creations and policy updates are tracked and deleted when you delete the configuration policy.
+When you modify an existing configuration policy, `pruneObjectBehavior` does not work. View the following reasons why `pruneObjectBehavior` might not work:
+
+- If you set `pruneObjectBehvior` to `DeleteAll` or `DeleteIfCreated`, old resources that were created before modifying are not cleaned correctly. Only new resources from policy creations and policy updates are tracked and deleted when you delete the configuration policy.
```
Comment (cadawson) [inline release_notes/known_issues_governance.adoc]:
```suggestion
- If you set `pruneObjectBehavior` to `DeleteAll` or `DeleteIfCreated` in a configuration policy, old resources that were created before modifying are not cleaned correctly. Only new resources from policy creations and policy updates are tracked and deleted when you delete the configuration policy.
```

File: release_notes/known_issues_governance.adoc (hunk 1 above)
Comment (cadawson) [inline release_notes/known_issues_governance.adoc]:
```suggestion
- If you set `pruneObjectBehavior` to `None` or do not set the parameter value, old objects might be unintentionally deleted on the managed cluster. Specifically, this occurs when you change the `name`, `namespace`, `kind`, or `apiversion` in the template. The parameter fields can dynamically change when the `object-templates-raw` or `nameSelector` parameters change. 
```

File: release_notes/known_issues_governance.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/known_issues_governance.adoc]: What are we modifying here? "...resources that you created before modifying the `pruneObjectBehavior` setting are not cleaned correctly"?

**Review by cadawson — CHANGES_REQUESTED** [review]
A couple of fixes and a question. Otherwise looks good. Thanks!

File: release_notes/known_issues_governance.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues_governance.adoc]: Great catch @yiraeChristineKim changing that now

Comment (dockerymick) [reply release_notes/known_issues_governance.adoc]: From my perspective, the parameter values for `pruneObjectBehavior` are `None`, `DeleteAll`, and `DeleteIfCreated`

File: release_notes/known_issues_governance.adoc (hunk 2 above)
Comment (dockerymick) [reply release_notes/known_issues_governance.adoc]: Modifying the configuration policy

Comment (dockerymick) [reply release_notes/known_issues_governance.adoc]: @cadawson I edited your suggestion, did that clarify things?

File: release_notes/known_issues_governance.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues_governance.adoc]: Thanks! I updated this

File: release_notes/known_issues_governance.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/known_issues_governance.adoc]: Yep. That works for me. Thanks!

File: release_notes/known_issues_governance.adoc (hunk 1 above)
Comment (cadawson) [reply release_notes/known_issues_governance.adoc]: @dockerymick: Please make sure you add the missing "a" to "Behavior".

**Review by yiraeChristineKim — APPROVED** [review]

File: release_notes/known_issues_governance.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues_governance.adoc]: Updated

**Comment by dockerymick** [comment]
@cadawson thanks for the reminder about the spelling. Are there any other changes or are you fine with me moving forward to merge?

**Review by cadawson — APPROVED** [review]
Looks good! Thanks!

**Comment by dockerymick** [comment]
@cadawson Thanks for your approval. Merging this and will make updates for 2.7

**Pull request merged**